public/static/app/perf-bb/SignalProcessingCtrl.test.js

```diff
@@ -34,7 +34,43 @@ describe('SignalProcessingFactoriesTest', () => {
       it('HIDDEN should be false', () => expect(unprocessed({ processed_type: PROCESSED_TYPE.HIDDEN })).toBe(false))
       it('ACKNOWLEDGED should be false', () => expect(unprocessed({ processed_type: PROCESSED_TYPE.ACKNOWLEDGED })).toBe(false))
     });
-
   });
 
-});
\ No newline at end of file
+  describe('SignalProcessingCtrl', () => {
+    let $controller;
+    let settings;
+
+    beforeEach(inject(function(_$controller_, _Settings_) {
+      $controller = _$controller_;
+      settings = _Settings_;
+    }));
+
+    describe('changePointFilter', () => {
+      let $scope;
+      let controller;
+
+      beforeEach(function() {
+        $scope = {};
+        controller = $controller('SignalProcessingCtrl', {$scope: $scope});
+      });
+
+      afterEach(function() {
+        settings.perf.signalProcessing.persistentFiltering = {};
+      });
+
+      it('should use defaults if no user preference specified', () => {
+        expect(controller.changePointFilter()).toEqual(controller.defaultChangePointFilter);
+      });
+
+      it('should use user preferences if specified', () => {
+        const filtering = {
+          create_time: ">now",
+          project: "=my-project",
+          probability: ">3.14",
```

Comment:
Have you also tested the code with 'local folder overrides' in chrome?

This reminds me that we should probably also validate that the value is > 0 and < 1 but one thing at a time.

Reply:
Yes, I tried it locally in chrome and it did what I would expect.

Extra validation is outside the scope of this ticket.

Reply:
:+1:  Any more test coverage is a win.